packages/lib/Logger.ts

```diff
@@ -287,6 +291,45 @@ class Logger {
 		}
 	}
 
+	private cleanLogFile(logFileFullPath) {
+		// eslint-disable-next-line promise/prefer-await-to-then
+		return Logger.fsDriver().stat(logFileFullPath).then(({ size }) => {
```

Comment:
Rather than disabling the [`promise/prefer-await-to-then` rule](https://github.com/eslint-community/eslint-plugin-promise/blob/main/docs/rules/prefer-await-to-then.md) for this line, consider making [`cleanLogFile`](https://github.com/laurent22/joplin/pull/8376/files#diff-1f533a0b60ed59ff38b3e902aebead3a0114471ca080c6acfbbb514125061dccR294) [`async`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function) and using an [`await`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await#description) here.